OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -811,6 +826,41 @@ static bool Team(S server, Connection conn, Session.Client client, string s)
 			}
 		}
 
+		static bool ClearPlayerSpawn(S server, Connection conn, Session.Client client, string s)
+		{
+			var spawnPoint = Exts.ParseIntegerInvariant(s);
+			if (spawnPoint == 0)
+				return true;
+
+			var existingClient = server.LobbyInfo.Clients.FirstOrDefault(cc => cc.SpawnPoint == spawnPoint);
+			if (client != existingClient && !client.IsAdmin)
+			{
+				server.SendOrderTo(conn, "Message", "Only admins can clear spawn points.");
+				return true;
+			}
+
+			// Clearing a selected spawn point removes the player
+			if (existingClient != null)
+			{
+				existingClient.SpawnPoint = 0;
+				if (existingClient.State == Session.ClientState.Ready)
+					existingClient.State = Session.ClientState.NotReady;
+
+				server.SyncLobbyClients();
+				return true;
+			}
+
```

Comment:
I was expecting to see a check here to make sure that the spawn wasn't locked to a player slot.

Test this by adding
```
LockSpawn: True
Spawn: 1
```
under `PlayerReference@Multi0:` in map.yaml for Fort Lonestar (our favorite map for exposing bugs in lobby/server PRs) or another map of your choice.